src/components/BaseButton.vue

```diff
@@ -70,6 +70,7 @@ defineProps<{
 
   &:disabled {
     color: var(--border-color) !important;
+    border: 1px solid var(--border-color);
```

Comment:
We already have this style on the button right? 🤔
![Mar-17-2023 17-38-12](https://user-images.githubusercontent.com/15967809/225900270-685c09ee-6a08-460d-9d05-6e850164d411.gif)

Reply:
Oh now I know why, ignore this